model/pipeline/build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply from: project(":").file("build_rules.gradle")
+applyJavaNature(enableFindbugs: false)
+applyGrpcNature()
```

Comment:
<!--new_thread; commit:f0b52c35fe2ad9680b8b41a35f95d8acb800da03; resolved:0-->
Where are the source files specified? Are they implied? How do we configure the output? Currently, our protos are actually put in the wrong place in the jar, really, and we should namespace them. How can we do that? And can we pull in protos from one module to another without having to pull in a jar? We should be able to build three dependent proto modules all to Python without touching any Java technology.